packages/accounts/src/msca/plugins/multi-owner/signer.ts

```diff
@@ -0,0 +1,46 @@
+import type { SignTypedDataParams, SmartAccountSigner } from "@alchemy/aa-core";
+import { toBytes } from "viem";
+import { MultiOwnerPlugin } from "./plugin.js";
+
+export class MultiOwnerPluginSigner
+  implements SmartAccountSigner<SmartAccountSigner>
+{
+  signerType: string;
+  inner: SmartAccountSigner<any>;
+  plugin: MultiOwnerPlugin;
+
+  // TODO(moldy) -- use a struct instead of args array
+  constructor(inner: SmartAccountSigner<any>, pluginAddress: string) {
+    this.signerType = inner.signerType;
```

Comment:
i think this should actually be something like `multi-owner:${inner.signerType}`